src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -399,10 +401,108 @@ void generateDeclarations() {
     }
   }
 
+  private boolean isGoogCall(Node node) {
+    if (!node.isCall()) {
```

Comment:
Closures uses `matchesQualifiedName` with a constant constructed as `IR.getprop(IR.name("goog"), IR.string("require"))`. Maybe that'd be a bit less wordy here?